### ISSUE TITLE
lru cache interferes with use of kwargs

### ISSUE THREAD

**Issue by Slater-Victoroff** [repoze/repoze.lru#24]
Passing kwargs into a function decorated by lru_cache throws an error:

`
@lru_cache(500)
def some_function(stuff, **kwargs):
    # do stuff
`

throws an error

**Comment by jorgeas80** [comment]
Yeah, I actually started using [django.utils.lru.cache](https://docs.djangoproject.com/en/1.9/releases/1.7/#django-utils-functional-memoize-function) instead. It works with kwargs

**Comment by erdnaxeli** [comment]
You can use the LRU cache from [the std lib](https://docs.python.org/3.5/library/functools.html#functools.lru_cache).
